src/components/admin/order/CreateRoute.jsx

```
@@ -77,6 +77,7 @@ const CreateRoute = () => {
     }
     const fetchDeliveryStaffOptions = async () => {
         try {
+            const response = await api.get("users")
             const response = await api.get("users")
```

Biome (1.9.4)

[error] Shouldn't redeclare 'response'. Consider to delete it or rename it.

'response' is defined here:


(lint/suspicious/noRedeclare)